backend/src/Break.cc

```diff
@@ -57,6 +58,7 @@ struct Defaults
                         30,
                         "",
                         150,
+                        true,
```

Comment:
You missed one default value. I just noticed that the 'quiet' default in 'struct Defaults' isn't used. You could use it on line 197. So 'def.quiet' instead of 'false'. But I also don't mind removing it from 'struct Defaults'.